### DISCUSSION TITLE
Ошибка во время compose up

### DISCUSSION THREAD

**Discussion by shilophrenic** [xtrime-ru/antizapret-vpn-docker#13] [Q&A] [answered]
Столкнулся со следующей проблемой
![1](https://github.com/user-attachments/assets/1768e8bf-b461-4aa7-a4bd-c9573b32cc84)

**Comment by xtrime-ru** [comment]
Какие то проблемы на стороне сервера knot-resolver. Проще всего попробовать попозже. 

Если проблема будет повторяться, то я запушу чуть попозже готовый образ в docker hub, можно будет его скачать.

**Reply by xtrime-ru** [reply, marked as answer]
Запушил образ, обновил доку. 

Загрузить и запустить образ без сборки:
```shell
docker compose pull && docker compose up -d
```
